resources/assets/helpers/index.js

```diff
@@ -1009,3 +1009,25 @@ export function getMillisecondsFromDays(days) {
   // @TODO make this more flexible, ie. get milliseconds from hours vs days etc
   return days * 1440 * 60 * 1000;
 }
+
+/**
+ * Checks if given pathname matches an entry in a paths config.
+ *
+ * @param {Array} paths
+ * @param {String} pathname
+ * @return {Boolean}
+ */
+export const isExcludedPath = (paths, pathname) => {
+  return paths.find(path => {
+    if (path.includes('*')) {
```

Comment:
Now that this lives in a separate helper (which I agree with @katiecrane is 👍 ), I think this function name seems a little less readable than it could be. It's checking whether the pathname matches one of the patterns in paths -- but it's the other components that call it which use it or the sake of excluding. One day we might want to use it for including something (vs excluding it) based on a path.

What do you think about refactoring to something like `isCurrentPathInPaths(paths)`, and DRY passing the ` window.location.pathname` as an argument?


```suggestion
/**
 * Checks if current path matches an item in given paths array.
 *
 * @param {Array} paths
 * @return {Boolean}
 */
export const isCurrentPathInPaths = (paths) => {
  const pathname = window.location.pathname;
    
  return paths.find(path => {
    if (path.includes('*')) {
```

Reply:
agree on the name change! and like the suggestion for setting the pathname var inside the helper